test/AllTests.agda

```diff
@@ -33,6 +33,7 @@ import LiteralPatterns
 import Issue92
 import HeightMirror
 import TransparentFun
+-- import StreamFusion TODO: cannot import b/c of --erased-cubical flag
```

Comment:
Can you add some references to "size inference issues" in Agda, either in the test file or simply as a comment to the PR?

Reply:
The reason why I cannot import it here is not because of the bad inference of sizes, but simply because it uses the `--erased-cubical` flag while `AllTests` doesn't, but adding `--erased-cubical` to `AllTests` is not possible either because it also imports files that use `--erase-record-parameters`.

Reply:
The proper thing to do here would be to have a separate file with tests using `--erased-cubical`